### ISSUE TITLE
enum body is deleted

### ISSUE THREAD

**Issue by r-arias** [thorbenvh8/prettier-java#33]
It looks like only the values of enums are preserved; other code in the enum's body gets deleted.

This:

```java
public enum Enum {

    THIS_IS_GOOD,
    THIS_IS_FINE;

    public static final String thisWillBeDeleted = "DELETED";

    public String toString() {
        return "STRING";
    }

}
```

turns into this:
```java
public enum Enum {

    THIS_IS_GOOD,
    THIS_IS_FINE;

}
```

**Comment by thorbenvh8** [comment]
thanks for the feedback and excellent example!

**Comment by thorbenvh8** [comment]
Fixed, had a TODO exactly on that position ;)
Fixed in version 0.0.3, hopefully soon
